aiida/common/escaping.py

```diff
@@ -40,6 +40,10 @@ def escape_for_bash(str_to_escape):
 
     str_to_escape = str(str_to_escape)
 
+    # dont espace when string contain $ symbol, usually this is for ENV variable
```

Comment:
Oh, this fix is way too far-reaching; it will affect any string that contains a dollar sign, not just the one we are looking at.

Can you provide a link to the issue that @giovannipizzi mentioned where they discuss the pros/cons of using single/double quotes?

Reply:
I would vote for option 2. But this prompt setup question should be in the computer setup stage just after the scheduler setting there, I assume.